config.example/group_vars/slurm-cluster.yml

```diff
@@ -36,6 +36,8 @@ hosts_add_ansible_managed_hosts_groups: ["slurm-cluster"]
 ################################################################################
 slurm_cluster_install_cuda: yes
 slurm_cluster_install_openmpi: yes
+slurm_cluster_install_nvidia_driver: yes
+slurm_cluster_install_singularity: yes
```

Comment:
@dholt : I go back and forth on whether Singularity should be enabled by default, given that we also enable enroot/pyxis by default... Any thoughts? Leaving it as is for now to minimize scope of this PR.

Reply:
Great question... based on some other PRs, it should probably be installed via Spack vs manually built, and I'm ok with setting it to not being built by default

Reply:
Ok, flipping this to "no".